docs/conf.py

```diff
@@ -1,13 +1,16 @@
-"""Sphinx configuration file for pytest-memray documentation."""
+"""Sphinx configuration file for pytest-memray documentation."""  # noqa: INP001
 from __future__ import annotations
 
 import sys
 from pathlib import Path
 from subprocess import check_output
+from typing import TYPE_CHECKING
 
-from sphinx.application import Sphinx
 from sphinxcontrib.programoutput import Command
 
+if TYPE_CHECKING:
+    from sphinx.application import Sphinx
```

Comment:
Why does this need to be moved below an `if TYPE_CHECKING:`?

Reply:
Done automatically by ruff, read https://beta.ruff.rs/docs/rules/typing-only-first-party-import/